cove/templates/500.html

```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends app_base_template %}
```

Comment:
There was something weird about how the the 500 view is loaded which did not give me that variable, so I had to do it this way.